code/modules/client/verbs/ooc.dm

```diff
@@ -468,3 +468,68 @@ ADMIN_VERB(reset_ooc_color, R_FUN, "Reset Player OOC Color", "Returns player OOC
 	set desc = "View the current map vote tally counts."
 	set category = "Server"
 	to_chat(mob, SSmap_vote.tally_printout)
+
+/client/var/forumlinklimit = 0
```

Comment:
We want these under the base /client definition because the LSP doesn't handle this well